### DISCUSSION TITLE
Disabling android release tasks for KMP

### DISCUSSION THREAD

**Discussion by mightyguava** [detekt/detekt#7189] [Q&A] [answered]
As of the latest release, the multiplatform tasks for detekt do not respect `ignoredBuildTypes` and `ignoredVariants`, so we get tasks created for release builds too. https://github.com/detekt/detekt/pull/7016 fixes this, but was skipped in the last release. I was wondering if there's plans to include it in a near future release, or will it only be available in 2.0.0?

**Comment by cortinico** [comment, marked as answer]
> I was wondering if there's plans to include it in a near future release, or will it only be available in 2.0.0?

Nope we're not planning to backport that as it's a significant refactoring of the detekt Gradle Plugin.
